Applications/ProductionManager/Contracts/Import/Order.cs

```diff
@@ -0,0 +1,48 @@
+﻿
+using Newtonsoft.Json;
+
+namespace HomagConnect.ProductionManager.Contracts.Import
+{
+    /// <summary>
+    /// Order data
+    /// </summary>
+    public class Order
+    {
+        /// <summary>
+        /// The unique id of this order
+        /// </summary>
+        [JsonProperty(Order = 1)]
+        public string Id { get; set; } = string.Empty;
+
+        /// <summary>
+        ///  The name of this order
+        /// </summary>
+        [JsonProperty(Order = 2)]
+        public string Name { get; set; } = string.Empty;
+
+        /// <summary>
+        /// The name of the customer of this order
+        /// </summary>
+        [JsonProperty(Order = 3)]
+        public string CustomerName { get; set; } = string.Empty;
+        /// <summary>
+        /// The date the order was created at
+        /// </summary>
+
+        [JsonProperty(Order = 4)]
+        public DateTimeOffset OrderDate { get; set; }
```

Comment:
Please clarify with @joBueche whether to use DateTime or DateTimeOffset. All datetimes should be returned in the TimeZone which is set in the subscription.

Reply:
I summarized my thoughts in a OneNote document. I will provide the link in internal chat